app/controllers/api/v1/accounts/webhooks_controller.rb

```diff
@@ -22,10 +22,15 @@ def destroy
 
   private
 
-  def webhook_params
+  def webhook_create_params
     params.require(:webhook).permit(:inbox_id, :name, :url, subscriptions: [])
   end
 
+  def webhook_update_params
+    # NOTE: Do not allow updating linked inbox to avoid events being sent for unexpected inboxes.
+    params.require(:webhook).permit(:name, :url, subscriptions: [])
+  end
```

Comment:
In my own fork, we also block updating the webhook URL.

The idea is that changing the URL/inbox is a critical action, which shouldn't be done lightly, so deleting and/or creating a new webhook is the way to go.

But as that might be disruptive to existing users, I decided on only blocking updating the new inbox field, but we might want to remove that as well.